docs/components/concepts/process-instance-migration.md

```diff
@@ -152,6 +152,7 @@ The following limitations exist that may be supported in future versions:
   - An element that becomes nested in a newly added sub-process
   - An element that was nested in a sub-process is no longer nested in that sub-process
 - Mapping instructions cannot change the element type
+- The process instance must be in a wait state, i.e. it may not be taking a sequence flow or triggering an event while migrating the instance
```

Comment:
```suggestion
- The process instance must be in a wait state, i.e. waiting for the job worker to activate/complete the job or user to complete a user task. It may not be taking a sequence flow or triggering an event while migrating the instance.
```

Reply:
That makes sense to me! I just wanted to have a clear info for the user in which situation they can migrate the instance 😄

Reply:
@christinaausley I'll open another PR with this adjustment 🙇